orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationAiModel.java

```diff
@@ -28,8 +94,27 @@ public class OrchestrationAiModel extends LLMModuleConfig {
   /** Azure OpenAI GPT-4o-mini chat completions model */
   public static final OrchestrationAiModel GPT_4O_MINI = new OrchestrationAiModel("gpt-4o-mini");
 
+  /** Google Cloud Platform Gemini 1.0 Pro model */
+  public static final OrchestrationAiModel GEMINI_1_0_PRO =
+      new OrchestrationAiModel("gemini-1.0-pro");
+
+  /** Google Cloud Platform Gemini 1.5 Pro model */
+  public static final OrchestrationAiModel GEMINI_1_5_PRO =
+      new OrchestrationAiModel("gemini-1.5-pro");
+
+  /** Google Cloud Platform Gemini 1.5 Flash model */
+  public static final OrchestrationAiModel GEMINI_1_5_FLASH =
+      new OrchestrationAiModel("gemini-1.5-flash");
+
   OrchestrationAiModel(@Nonnull final String modelName) {
-    setModelName(modelName);
-    setModelParams(Map.of());
+    this(modelName, Map.of(), "latest");
```

Comment:
(Question)

Is `latest` always supported by orchestration?

Reply:
from my testing and understanding, yes